verify.py

```diff
@@ -1,28 +1,55 @@
+import argparse
+import pathlib
 import base64
 import json
+import os
 
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import padding
 
-sig = base64.b64decode("base64EncodedSignature==")
+from PIL import Image
+from pyzbar import pyzbar
 
-payload = '{"id":"01/IL/ABCD1234ABCD1234ABCD1234ABCD1234#ABCD1234","et":1,"ct":1,"c":"IL MOH","cn":null,"fn":null,"g":null,"f":null,"gl":null,"fl":null,"idp":null,"idl":null,"b":"0001-01-01","e":"0001-01-01","a":"0001-01-01","p":[{"idl":"0123456789","e":"2021-01-01"}]}'
 
-h = hashes.Hash(hashes.SHA256())
-h.update(payload.encode("utf-8"))
-digest = h.finalize()
+def cert(name):
+    this_dir = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(this_dir, 'certs', name)
```

Comment:
A nicer approach would be to use `pathlib` for concat'ing paths

Reply:
Fixed